packages/voting-contract/src/lib.rs

```diff
@@ -157,7 +162,8 @@ where
     P: Serialize + DeserializeOwned,
 {
     let mut proposal = proposals::<P>().load(storage, proposal_id)?;
-
+    // Update Status
+    proposal.update_status(&env.block);
```

Comment:
This is not needed. Status will be updated below, and the check is already using `current_status`.

Reply:
Funny how we have two or three different implementations / variations of the same logic.

It will be good to move this `voting-contract` package to cw-plus at some point, and change all multisigs (cw-plus, poe-contracts, and Tgrade-contracts) to use it.

Reply:
Created See https://github.com/confio/poe-contracts/issues/194 to track that effort.